src/fragments/lib/logging/android/setup_logging/30_initialize_without_config.mdx

```diff
@@ -0,0 +1 @@
+The `<log-group-name>` and `region` is the value you specified in the CDK construct as part of provisioning your AWS backend resources.  The configuration file is the data source for the logging plugin to know where, when and what to send logs.  The example below configures the logging plugin to automatically send all logs at log level `ERROR` at 60 seconds interval and store logs locally up to 1MB.
```

Comment:
can we maybe also show in the code sample an example of the configurations that you can change in code?

Reply:
removing this page since it's a placeholder for Android.